.circleci/config.yml

```diff
@@ -109,7 +105,6 @@ jobs:
             npm publish --access=public
 
 workflows:
-  version: 2
```

Comment:
does versioning workflows matter on circle ci?

Reply:
Not anymore, that was removed in 2.1: https://discuss.circleci.com/t/circleci-2-1-config-overview/26057